doc/userguide/rules/lua-detection.rst

```diff
@@ -15,7 +15,8 @@ Syntax:
 
 The script filename will be appended to your default rules location.
 
-The script has 2 parts, an init function and a match function. First, the init.
+The script has 2 parts, an init function and a match function. First, the init.  
```

Comment:
this sentence makes less sense now: the "init" block is supposed to follow it directly I think